src/Personal/reducer.js

```diff
@@ -0,0 +1,62 @@
+import { actionType } from './actionCreator';
+
+const initialBackup = {
+  response: 'Click button to backup data',
+};
+
+const initialUser = {
+  username: '',
+  email: '',
+  accessToken: '0xa143981f3ec758296a1575146eab03ef047b7e40',
+};
+
+const backupState = (state = initialBackup, action) => {
```

Comment:
When do update in states via reducer, it would be better to do it in this way: return { ...state, xxx: 'new value' }, so later when you add new state fields, you won't need to change previous implementations.